test/conftest.py

```diff
@@ -0,0 +1,43 @@
+"""Pytest configurations for multinet tests."""
```

Comment:
Is `conftest.py` a standard name for a module that defines PyTest fixtures?

Reply:
Yes https://stackoverflow.com/questions/34466027/in-pytest-what-is-the-use-of-conftest-py-files